services/src/datasets/external/gfbio.rs

```diff
@@ -114,7 +86,7 @@ impl GfbioDataProvider {
         let pool = Pool::builder().build(pg_mgr).await?;
 
         let (column_hash_to_name, column_name_to_hash) =
-            Self::resolve_columns(pool.get().await?, &db_config.schema).await?;
+            Self::resolve_columns(&pool.get().await?, &db_config.schema).await?;
 
```

Comment:
what takes super long? The listing is fast for me, only adding the layers takes some time. This could be optimized in the frontend by parallelizing and reducing redundant calls (which is quite some work I suspect).

Reply:
1) 

![image](https://user-images.githubusercontent.com/4947061/211203638-8b94fb8e-2e50-4312-ba27-8ff9492ae8c0.png)

This call takes 30 Seconds. This does not happen if I run it without the NFDI flag.

2) Yeah, your solution sounds reasonable if we don't have this implemented. On the other hand, why don't we just output an empty list?

Reply:
1. is fixed on your end now, right?
2. because, then the provider would be visible in the datasets list without providing any functionality.